qiskit/primitives/base/base_primitive.py

```diff
@@ -104,7 +104,7 @@ def _validate_parameter_values(
 
     @staticmethod
     def _cross_validate_circuits_parameter_values(
-        circuits: tuple[QuantumCircuit, ...], parameter_values: tuple[tuple[float, ...], ...]
+        circuits: list[QuantumCircuit, ...], parameter_values: tuple[tuple[float, ...], ...]
```

Comment:
I'm not sure why you change tuple with list. Anyways, I suggest the following. How about `Sequence` to allow both list and tuple?
```suggestion
        circuits: list[QuantumCircuit], parameter_values: tuple[tuple[float, ...], ...]
```

Reply:
The purpose is to change the element. Anyway, thank you for pointing out my wrong type hint.